src/cpu/core_dyn_x86/decoder.h

```diff
@@ -2896,7 +2896,9 @@ static CacheBlock * CreateCacheBlock(CodePageHandler * codepage,PhysPt start,Bit
 	/* Setup the correct end-address */
 	decode.active_block->page.end=--decode.page.index;
 	dyn_mem_execute(cache_addr, cache_bytes);
-	dyn_cache_invalidate(cache_addr, cache_bytes);
+	const size_t cache_flush_bytes = static_cast<size_t>(decode.block->cache.size);
```

Comment:
(very minor), maybe `const auto`, given the RHS is cast? (same with a couple places below)